<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="createGraphButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>131</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать граф</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="gbLandmarksBasic">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>110</y>
      <width>248</width>
      <height>156</height>
     </rect>
    </property>
    <property name="title">
     <string>Landmarks Basic</string>
    </property>
    <widget class="QComboBox" name="cmbBasicMethod">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>highest-degree</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>best-coverage</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>random</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="txtBasicK">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtBasicM">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCreateBasic">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>100</y>
       <width>88</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Создать</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbLandmarksBFS">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>238</width>
      <height>156</height>
     </rect>
    </property>
    <property name="title">
     <string>Landmarks BFS</string>
    </property>
    <widget class="QComboBox" name="cmbBFSMethod">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>highest-degree</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>best-coverage</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>random</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="txtBFSK">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtBFSM">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnCreateBFS">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>88</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Создать</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gbCalculations">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>391</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Вычисления</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLineEdit" name="txtS">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="txtT">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QCheckBox" name="chkInternalRepr">
       <property name="text">
        <string>Исп-ть внут-нее пред-ние</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btnExact">
       <property name="text">
        <string>Точный алгоритм</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" rowspan="3">
      <widget class="QTextEdit" name="txtResults">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="btnApproxBFS">
       <property name="text">
        <string>Landmarks BFS</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="btnApproxBasic">
       <property name="text">
        <string>Landmarks Basic</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
